test/colab/test.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -eo pipefail
+
+trap 'wait;' SIGINT SIGTERM
+
+. $HOME/.poetry/env
+poetry build --format wheel
```

Comment:
Can we just mount the wheel in the container and do the build ourside? Feels odd to do it inside the container.

Reply:
I don't think the python version used to build the wheel matters. We build it with 3.8 when we push to pypi and that works on all python versions we support.